### PR TITLE
A11y ford

### PR DESCRIPTION
**Pull request by elopezanaya**
## 3226191 - Issue 2 – Elements must only use allowed ARIA attributes -

Removing aria attributes and title for chat button, relying on text on display

image

## 3226201 : Tittle attributes used as labels for buttons
 remove title attributes for :  
- minimize button
- close button

image

- download chat
- email
- sound

image

image

image

upload file and send button are not in our control.

### CODE REVIEW

File: chat-components/src/components/chatbutton/common/defaultProps/defaultChatButtonControlProps.ts [hunk 1]
```diff
@@ -5,10 +5,7 @@ export const defaultChatButtonControlProps: IChatButtonControlProps = {
     id: Ids.DefaultChatButtonId,
     dir: "ltr",
     role: "button",
-    ariaLabel: AriaLabels.LetsChatWeAreOnline,
```
Comment (charliewang95) [inline chat-components/src/components/chatbutton/common/defaultProps/defaultChatButtonControlProps.ts:8]: Why are these removed?

File: chat-components/src/components/common/commandbutton/CommandButton.tsx [hunk 2]
```diff
@@ -66,6 +66,7 @@ function CommandButton(props: ICommandButtonProps) {
                     disabled={props.disabled}
                     className={props.className}
                     styles={buttonStyles}
+                    title={props.hideButtonTitle ? undefined : props.ariaLabel}
```
Comment (charliewang95) [inline chat-components/src/components/common/commandbutton/CommandButton.tsx:69]: Why are we using title not ariaLabel?

File: chat-components/src/components/chatbutton/common/defaultProps/defaultChatButtonControlProps.ts (hunk 1 above)
Comment (elopezanaya) [reply chat-components/src/components/chatbutton/common/defaultProps/defaultChatButtonControlProps.ts:8]: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3226191 , 

the ask is to remove title and aria label

File: chat-components/src/components/common/commandbutton/CommandButton.tsx (hunk 2 above)
Comment (elopezanaya) [reply chat-components/src/components/common/commandbutton/CommandButton.tsx:69]: I can  add the aria label, but for what I see , the aria label is generally set at specific component level (Email, download, etc)

**Review by charliewang95 — APPROVED** [review]

**Review by xTEddie — APPROVED** [review]
